### PR TITLE
Feature/FOUR-14386: Update Cards (UI updated).

### PR DESCRIPTION
**Pull request by CarliPinell**
## Solution
- Go to Processes menu
- Navigate through cards

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-14386

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

### CODE REVIEW

**Review by gustavobascope — APPROVED** [review]
Approved

File: resources/js/processes-catalogue/components/utils/Card.vue [hunk 1]
```diff
@@ -39,21 +40,39 @@
       />
     </b-card-text>
   </b-card>
+  <b-card v-else
+  class="text-center d-flex align-items-center justify-content-center card-process2">
+    <span v-if="cardMessage === 'show-page'">Page {{ currentPage }} of {{ totalPages }}</span>
+    <span v-if="cardMessage === 'show-more' && !loading"> {{ $t('Show More') }}</span>
```
Comment (gproly) [inline resources/js/processes-catalogue/components/utils/Card.vue:46]: Please add missing translations into en.json file

File: resources/js/processes-catalogue/components/utils/Card.vue [hunk 2]
```diff
@@ -39,21 +40,39 @@
...
+    <span v-if="loading"><i class="fas fa-spinner fa-spin"></i> {{ $t('Loading') }}...</span>
```
Comment (gproly) [inline resources/js/processes-catalogue/components/utils/Card.vue:47]: Please add missing translations into en.json file

**Review by gproly — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/utils/Card.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/utils/Card.vue:46]: Done

File: resources/js/processes-catalogue/components/utils/Card.vue (hunk 2 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/utils/Card.vue:47]: Done

**Review by gproly — APPROVED** [review]

**Pull request merged**